tests/testthat/test-reorder.R

```diff
@@ -74,6 +74,23 @@ test_that("fct_infreq respects missing values", {
   expect_equal(levels(fct_infreq(f)), c(NA, "b", "a"))
 })
 
+test_that("fct_infreq() respects weights", {
+  x <- c("a", "b", "b", "c")
+  f <- factor(x, exclude = FALSE)
```

Comment:
```suggestion
  f <- factor(x)
```

Is there a reason for `exclude`? Or use `fct()`?

Reply:
Oops